### PR TITLE
Migration to @react-native-community/art

### PR DESCRIPTION
**Pull request by jomla97**
Migration from `react-native/ART` to `@react-native-community/art`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React, { PureComponent } from 'react';
-import { View, StyleSheet, ART, Text } from 'react-native';
+import { View, StyleSheet, Text } from 'react-native';
 import PropTypes from 'prop-types';
 
 import barcodes from 'jsbarcode/src/barcodes';
 
-const { Surface, Shape } = ART;
+import {Surface, Shape} from '@react-native-community/art';
```
Comment (wonsikin) [inline index.js]: @jomla97 Should we also update the package.json?

Comment (jomla97) [reply index.js]: Good call @wonsikin , done!

**Comment by wonsikin** [comment]
#52

**Comment by tmaly1980** [comment]
Should the version be bumped and npm updated?

**Comment by jomla97** [comment]
Absolutely @tmaly1980 is there anything else to push with the same update? Should i commit a change to the version? If so, should this be considered a minor or patch version bump?

**Comment by wonsikin** [comment]
#46

**Comment by wonsikin** [comment]
#50

**Pull request merged**